tests/Feature/SitemapTest.php

```
@@ -64,3 +64,26 @@
     $response->assertOk()
         ->assertSee($expectedUpdatesEntry, false);
 });
+
+it('omits lastmod from static pages while preserving announcement modification dates', function () {
+    Carbon::setTestNow(Carbon::create(2026, 5, 6, 12, 0, 0));
+
+    $announcement = Announcement::create([
```

PHPStan (2.2.8)

[warning] Call to an undefined static method App\Models\Announcement::create().


(staticMethod.notFound)